<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{border-radius:2px;background:rgb(39,140,222);}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{color:white}</string>
    </property>
    <property name="text">
     <string>查找</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_2{color:white;font-size:18px;}
#label_2:hover{background:rgb(255,255,255,50%);}</string>
    </property>
    <property name="text">
     <string>—</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_3{color:white;font-size:20px;}
#label_3:hover{background:rgb(255,255,255,50%);}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>35</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_5{font-weight:bold;font-size:16px;}
#label_5:hover{color:rgb(255,255,255);}</string>
    </property>
    <property name="text">
     <string>找群</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>35</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_4{font-weight:bold;font-size:16px;}
#label_4:hover{color:rgb(255,255,255);}</string>
    </property>
    <property name="text">
     <string>找人</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2{background:rgb(235,242,249);}</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit{background:white;border: 1px solid rgb(60,163,247);border-radius:2px;}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton{background:rgb(60,163,247);color:white;border-radius:2px}
#pushButton:hover{background:rgb(50,163,240,50%);}</string>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_6{background:rgb(233,237,238);color:rgb(104,104,104);font-weight:normal;}</string>
    </property>
    <property name="text">
     <string>搜索结果为：</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>155</y>
      <width>400</width>
      <height>145</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_3{background:rgb(245,245,245);border-radius:2px;}</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>35</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_7{border-image:url(&quot;:/headportrait/cloud.jpg&quot;);border-radius:37px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_8{font-weight:bold;font-size:18px;}
#label_8:hover{color:black;}</string>
     </property>
     <property name="text">
      <string>借我</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>75</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_9{font-weight:normal;font-size:15px;}
#label_9:hover{color:black;}</string>
     </property>
     <property name="text">
      <string>编辑个性签名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_2{color:white;background:rgb(143,192,231);border-radius:2px;}
#pushButton_2:hover{background:rgb(89,169,236);}</string>
     </property>
     <property name="text">
      <string>+好友</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
